### PR TITLE
Mostly an Update ru.m3u

### PR DESCRIPTION
**Pull request by AntiPontifex**
See title

### CODE REVIEW

File: streams/ru.m3u [hunk 1]
```diff
@@ -47,13 +47,13 @@ https://hls-03-video.webcaramba.com/rutv/live.m3u8
...
 http://serv30.vintera.tv:8081/sochi/sochi24_tv/playlist.m3u8
-#EXTINF:-1 tvg-id="",Sochi Live HD (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="SochiLiveHD.ru",Sochi Live HD (720p) [Not 24/7]
 http://serv30.vintera.tv:8081/sochi/sochi_stream/playlist.m3u8
 #EXTINF:-1 tvg-id="StarFamily.ru",Star Family
 http://89.254.128.5:22280/star_family_hd/video.m3u8
 #EXTINF:-1 tvg-id="",TRK 555 (720p)
 http://trk555.tv:8888/live
-#EXTINF:-1 tvg-id="",TVOE TV HD
+#EXTINF:-1 tvg-id="TvoyeTVYumor.ru",TVOE TV HD
```
Comment (Dum4G) [inline streams/ru.m3u:56]: bro you can make it I believe

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/ru.m3u (hunk 1 above)
Comment (BellezaEmporium) [reply streams/ru.m3u:56]: I know why, the original TVoe isn't in the database, lol.

Comment (BellezaEmporium) [reply streams/ru.m3u:56]: The correct TVOE Humor btw : https://tvoetv.space/tvoetv_humor/video.m3u8

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
On hold until we have TVoe added on the database.

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**